VirtualList/useThemeVirtualList.js

```diff
@@ -133,7 +133,7 @@ const useSpottable = (props, instances) => {
 	useEffect(() => {
 		return () => {
 			// TODO: Fix eslint
-			pause.resume(); // eslint-disable-line react-hooks/exhaustive-deps
+			pause.resume();
```

Comment:
@sjro Could you please remove the comment above this line also?

Reply:
ok